freightliner_cascadia_2012_dot_80550/drivers.launch.py

```diff
@@ -67,14 +67,14 @@ def generate_launch_description():
         ]
     )
 
-    dsrc_group = GroupAction(
-        condition=IfCondition(PythonExpression(["'dsrc_driver' in '", drivers, "'.split()"])),
+    v2x_ros_driver = GroupAction(
```

Comment:
Let's rename to `v2x_driver_group` to maintain consistency with other vehicle configs